recipes/broh5/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "broh5" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/broh5-{{ version }}.tar.gz
+  sha256: 3f6fee7f2ea065690d6669a404093bdec17793ac1e6a6631912f6bf57cd3bcd6
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - broh5 = broh5.main:main
+
+requirements:
+  host:    
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy >1.21
+    - h5py >3.7
+    - pillow
+    - matplotlib-base
+    - nicegui >1.3.16
+
+test:
+  imports:
+    - broh5
```

Comment:
```suggestion
test:
  imports:
    - broh5
  commands:
    - pip check
    - broh5 --help
  requires:
    - pip
```

Reply:
It's a GUI app not CIL, so I remove "broh5 --help"

Reply:
We need some way to test the entry-point. Is there some `broh5 --version`?

Reply:
Good suggestion. Will implement that, but hope it's ok for this version.

Reply:
If you don't have it, no need to implement. I was just wondering how we can test if that entry-point is working. If you are the upstream dev and you are certain that calling it will yield the expected result, let's merge.